test/unit/gateways/orbital_test.rb

```diff
@@ -1953,4 +2000,11 @@ def post_scrubbed_echeck
       Conn close
     REQUEST
   end
+
+  def assert_xml_valid_to_xsd(data)
+    schema_file = File.read("#{File.dirname(__FILE__)}/../../schema/orbital/Request_PTI95.xsd")
+    doc = Nokogiri::XML(data)
+    xsd = Nokogiri::XML::Schema(schema_file)
```

Comment:
Test improvement suggestion here, can we memoize the reading/loading of the XSD so that it is only done once for the whole test file? This will speed up tests.

Reply:
Done!